pkg/controllers/workgenerator/manifests/resourceQuota.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
```

Comment:
Hi Ryan! Just a nit: this folder seems to have names of all three cases: snake, kebab, and camel.

Reply:
the files start with "test-" are actually applied to the k8s while other 2 files are just used to compare in the test.